packages/trader/src/Modules/Trading/Components/Form/TradeParams/strike.tsx

```diff
@@ -121,7 +122,7 @@ const Strike = observer(() => {
             </DesktopWrapper>
             <MobileWrapper>
                 <div className='mobile-widget__wrapper'>
-                    <div className='strike-widget' onClick={toggleWidget}>
+                    <div className='strike-widget' onClick={toggleWidget} onKeyDown={toggleWidget}>
                         {should_show_spot && (
                             <div className='mobile-widget__spot'>{<Text size='xs'>{localize('Spot')}</Text>}</div>
```

Comment:
Not exactly related to this PR, but do we have to  use <Localize/> rather then localize()

Reply:
Yes, indeed, it should be a preferred way 👍🏻